apps/web/app/store/useDateStore.ts

```diff
@@ -55,19 +67,19 @@ export const useDateStore = create<DateStore>((set) => ({
   // 월만 업데이트하는 함수
   setSelectedMonth: (month) => {
     set((state) => {
-      const newDate = new Date(state.selectedDate.year, month - 1, state.selectedDate.day);
-      const dayOfWeek = getDayOfWeek(newDate);
+      const { year, day } = state.selectedDate;
```

Comment:
```suggestion
      if (state.selectedDate.month === month) return state;
      const { year, day } = state.selectedDate;
```
예외처리 추가한다면 이런너낌도 괜찮을거같아

Reply:
그 유틸함수 같은건 const 로 했는데 그냥 선언형으로 통일하는게 좋을까?

혹시 위의 예외처리가 어떤 장점이 있는지 알 수 있을까?? (단순 궁금)

Reply:
개인적으로는 const로 통일해서 사용해서 재정의되는걸 방지하는게 나을거같은데... 사실 그냥 컨벤션 차이인듯해 export function export const 혼용되어있으면 일관성이 없어보일거같아서

Reply:
예외처리한다면 같은 month로 이동될때 리렌더링을 줄이는 너낌?

Reply:
아하 오키오키 수정할게~

Reply:
흠 나는 이제까지 유틸함수는 const / 컴포넌트는 선언형 함수로 써왔는데, 크게 문제 없으면 이렇게 사용해도 될까?